### ISSUE TITLE
Module not found: Error: Can't resolve '../../../../@embroider/externals/jquery'

### ISSUE THREAD

**Issue by piotrpalek** [embroider-build/embroider#170]
After resolving the issues in #169 I still have this next one. I'm trying to dig into what is happening here, but this is pretty foreign territory to me :)

<details>
<summary>Stack trace</summary>
<p>

```
=================================================================================

ENV Summary:

  TIME: Mon May 13 2019 22:32:37 GMT+0200 (CEST)
  TITLE: ember
  ARGV:
  - /Users/user/.nvm/versions/node/v8.9.1/bin/node
  - /Users/user/dev/test-app/node_modules/.bin/ember
  - build
  EXEC_PATH: /Users/user/.nvm/versions/node/v8.9.1/bin/node
  TMPDIR: /var/folders/3_/j1n1p4dn6gx884nt498fg9040000gn/T
  SHELL: /bin/zsh
  PATH:
  - /var/folders/3_/j1n1p4dn6gx884nt498fg9040000gn/T/yarn--1557779539159-0.9583461741250754
  - /Users/user/dev/test-app/node_modules/.bin
  - /Users/user/.config/yarn/link/node_modules/.bin
  - /Users/user/.nvm/versions/node/v8.9.1/libexec/lib/node_modules/npm/bin/node-gyp-bin
  - /Users/user/.nvm/versions/node/v8.9.1/lib/node_modules/npm/bin/node-gyp-bin
  - /Users/user/.nvm/versions/node/v8.9.1/bin/node_modules/npm/bin/node-gyp-bin
  - /Users/user/.nvm/versions/node/v8.9.1/bin
  - /usr/local/bin
  - /usr/bin
  - /bin
  - /usr/sbin
  - /sbin
  - /Users/user/.nvm/versions/node/v8.9.1/bin
  - /Users/user/go/bin
  - /Users/user/go/bin
  PLATFORM: darwin x64
  FREEMEM: 1352777728
  TOTALMEM: 17179869184
  UPTIME: 51820
  LOADAVG: 2.24072265625,1.98095703125,2.03564453125
  CPUS:
  - Intel(R) Core(TM) i7-4980HQ CPU @ 2.80GHz - 2800
  - Intel(R) Core(TM) i7-4980HQ CPU @ 2.80GHz - 2800
  - Intel(R) Core(TM) i7-4980HQ CPU @ 2.80GHz - 2800
  - Intel(R) Core(TM) i7-4980HQ CPU @ 2.80GHz - 2800
  - Intel(R) Core(TM) i7-4980HQ CPU @ 2.80GHz - 2800
  - Intel(R) Core(TM) i7-4980HQ CPU @ 2.80GHz - 2800
  - Intel(R) Core(TM) i7-4980HQ CPU @ 2.80GHz - 2800
  - Intel(R) Core(TM) i7-4980HQ CPU @ 2.80GHz - 2800
  ENDIANNESS: LE
  VERSIONS:
  - ares: 1.10.1-DEV
  - cldr: 31.0.1
  - http_parser: 2.7.0
  - icu: 59.1
  - modules: 57
  - nghttp2: 1.25.0
  - node: 8.9.1
  - openssl: 1.0.2m
  - tz: 2017b
  - unicode: 9.0
  - uv: 1.15.0
  - v8: 6.1.534.47
  - zlib: 1.2.11

ERROR Summary:

  - broccoliBuilderErrorStack: ModuleNotFoundError: Module not found: Error: Can't resolve '../../../../@embroider/externals/jquery' in '/private/var/folders/3_/j1n1p4dn6gx884nt498fg9040000gn/T/embroider/657b0b/node_modules/@ember/test-helpers/ember-test-helpers/legacy-0-6-x'
    at factory.create (/Users/user/dev/test-app/node_modules/webpack/lib/Compilation.js:821:10)
    at factory (/Users/user/dev/test-app/node_modules/webpack/lib/NormalModuleFactory.js:397:22)
    at resolver (/Users/user/dev/test-app/node_modules/webpack/lib/NormalModuleFactory.js:130:21)
    at asyncLib.parallel (/Users/user/dev/test-app/node_modules/webpack/lib/NormalModuleFactory.js:224:22)
    at /Users/user/dev/test-app/node_modules/neo-async/async.js:2825:7
    at /Users/user/dev/test-app/node_modules/neo-async/async.js:6886:13
    at normalResolver.resolve (/Users/user/dev/test-app/node_modules/webpack/lib/NormalModuleFactory.js:214:25)
    at doResolve (/Users/user/dev/test-app/node_modules/enhanced-resolve/lib/Resolver.js:184:12)
    at hook.callAsync (/Users/user/dev/test-app/node_modules/enhanced-resolve/lib/Resolver.js:238:5)
    at _fn0 (eval at create (/Users/user/dev/test-app/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:15:1)
  - code: [undefined]
  - codeFrame: Module not found: Error: Can't resolve '../../../../@embroider/externals/jquery' in '/private/var/folders/3_/j1n1p4dn6gx884nt498fg9040000gn/T/embroider/657b0b/node_modules/@ember/test-helpers/ember-test-helpers/legacy-0-6-x'
  - errorMessage: Module not found: Error: Can't resolve '../../../../@embroider/externals/jquery' in '/private/var/folders/3_/j1n1p4dn6gx884nt498fg9040000gn/T/embroider/657b0b/node_modules/@ember/test-helpers/ember-test-helpers/legacy-0-6-x'
        at PackagerRunner (@embroider/webpack)
-~- created here: -~-
    at new Plugin (/Users/user/dev/test-app/node_modules/broccoli-plugin/index.js:7:31)
    at new PackagerRunner (/Users/user/dev/test-app/node_modules/@embroider/core/src/to-broccoli-plugin.js:11:13)
    at Object.defaultPipeline [as compatBuild] (/Users/user/dev/test-app/node_modules/@embroider/compat/src/default-pipeline.js:30:12)
    at module.exports (/Users/user/dev/test-app/ember-cli-build.js:43:39)
    at Builder.readBuildFile (/Users/user/dev/test-app/node_modules/ember-cli/lib/models/builder.js:52:14)
    at Builder.setupBroccoliBuilder (/Users/user/dev/test-app/node_modules/ember-cli/lib/models/builder.js:66:22)
    at new Builder (/Users/user/dev/test-app/node_modules/ember-cli/lib/models/builder.js:32:10)
    at BuildTask.run (/Users/user/dev/test-app/node_modules/ember-cli/lib/tasks/build.js:15:19)
    at Promise.resolve.then (/Users/user/dev/test-app/node_modules/ember-cli/lib/models/command.js:243:46)
    at tryCatcher (/Users/user/dev/test-app/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:323:19)
-~- (end) -~-
  - errorType: Build Error
  - location:
    - column: [undefined]
    - file: [undefined]
    - line: [undefined]
    - treeDir: [undefined]
  - message: Module not found: Error: Can't resolve '../../../../@embroider/externals/jquery' in '/private/var/folders/3_/j1n1p4dn6gx884nt498fg9040000gn/T/embroider/657b0b/node_modules/@ember/test-helpers/ember-test-helpers/legacy-0-6-x'
        at PackagerRunner (@embroider/webpack)
-~- created here: -~-
    at new Plugin (/Users/user/dev/test-app/node_modules/broccoli-plugin/index.js:7:31)
    at new PackagerRunner (/Users/user/dev/test-app/node_modules/@embroider/core/src/to-broccoli-plugin.js:11:13)
    at Object.defaultPipeline [as compatBuild] (/Users/user/dev/test-app/node_modules/@embroider/compat/src/default-pipeline.js:30:12)
    at module.exports (/Users/user/dev/test-app/ember-cli-build.js:43:39)
    at Builder.readBuildFile (/Users/user/dev/test-app/node_modules/ember-cli/lib/models/builder.js:52:14)
    at Builder.setupBroccoliBuilder (/Users/user/dev/test-app/node_modules/ember-cli/lib/models/builder.js:66:22)
    at new Builder (/Users/user/dev/test-app/node_modules/ember-cli/lib/models/builder.js:32:10)
    at BuildTask.run (/Users/user/dev/test-app/node_modules/ember-cli/lib/tasks/build.js:15:19)
    at Promise.resolve.then (/Users/user/dev/test-app/node_modules/ember-cli/lib/models/command.js:243:46)
    at tryCatcher (/Users/user/dev/test-app/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:323:19)
-~- (end) -~-
  - name: BuildError
  - nodeAnnotation: @embroider/webpack
  - nodeName: PackagerRunner
  - originalErrorMessage: Module not found: Error: Can't resolve '../../../../@embroider/externals/jquery' in '/private/var/folders/3_/j1n1p4dn6gx884nt498fg9040000gn/T/embroider/657b0b/node_modules/@ember/test-helpers/ember-test-helpers/legacy-0-6-x'
  - stack: ModuleNotFoundError: Module not found: Error: Can't resolve '../../../../@embroider/externals/jquery' in '/private/var/folders/3_/j1n1p4dn6gx884nt498fg9040000gn/T/embroider/657b0b/node_modules/@ember/test-helpers/ember-test-helpers/legacy-0-6-x'
    at factory.create (/Users/user/dev/test-app/node_modules/webpack/lib/Compilation.js:821:10)
    at factory (/Users/user/dev/test-app/node_modules/webpack/lib/NormalModuleFactory.js:397:22)
    at resolver (/Users/user/dev/test-app/node_modules/webpack/lib/NormalModuleFactory.js:130:21)
    at asyncLib.parallel (/Users/user/dev/test-app/node_modules/webpack/lib/NormalModuleFactory.js:224:22)
    at /Users/user/dev/test-app/node_modules/neo-async/async.js:2825:7
    at /Users/user/dev/test-app/node_modules/neo-async/async.js:6886:13
    at normalResolver.resolve (/Users/user/dev/test-app/node_modules/webpack/lib/NormalModuleFactory.js:214:25)
    at doResolve (/Users/user/dev/test-app/node_modules/enhanced-resolve/lib/Resolver.js:184:12)
    at hook.callAsync (/Users/user/dev/test-app/node_modules/enhanced-resolve/lib/Resolver.js:238:5)
    at _fn0 (eval at create (/Users/user/dev/test-app/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:15:1)

=================================================================================
```

</p>
</details>

**Comment by ef4** [comment]
You are possibly effected by a caching bug that is fixed on master but not released.

Does `rm -rf $TMPDIR/embroider` fix that error?

**Comment by piotrpalek** [comment]
Yes it seems I have to run that prior to every build, but it helps.

**Comment by ef4** [comment]
This was fixed in #166.
